packages/anzen/src/__tests__/anzen_test.ts

```diff
@@ -393,17 +394,19 @@ describe('Result', () => {
       assert.equal(res.isSuccess, true);
       assert.equal(res.isFailure, false);
       assert.deepEqual(results, [1, 2, 'A']);
-      type checks = [
-        Expect<
-          Equal<
-            typeof res,
-            AnzenResultSuccess<[number, number, string]>
-          >
-        >,
-        Expect<
-          Equal<typeof results, [number, number, string]>
-        >,
-      ];
+      expectAll<
+        [
+          Expect<
+            Equal<
+              typeof res,
+              AnzenResultSuccess<[number, number, string]>
+            >
+          >,
+          Expect<
+            Equal<typeof results, [number, number, string]>
+          >,
+        ]
+      >();
```

Comment:
Here, I kept the `Expect<>` type for each value so that the wiggly line appears in each expect statement and not in the entire array (less desirable).